hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieMetricsPrometheusConfig.java

```diff
@@ -58,7 +58,7 @@
       .key(PUSHGATEWAY_PREFIX + ".delete.on.shutdown")
       .defaultValue(true)
       .sinceVersion("0.6.0")
-      .withDocumentation("");
+      .withDocumentation("Whether to delete the monitoring information from pushgateway at the end of the task, default true.");
```

Comment:
"Delete the pushgateway info or not when job shutdown, true by default" ?

Reply:
done